libs/payments/capability/project.json

```diff
@@ -12,7 +12,7 @@
         "outputPath": "dist/libs/payments/capability",
         "tsConfig": "libs/payments/capability/tsconfig.lib.json",
         "packageJson": "libs/payments/capability/package.json",
-        "main": "libs/payments/capability/src/index.ts",
```

Comment:
This change is interesting. Do you know why this has to change?

Reply:
This was a hacky fix, to fix an issue with the built version of the package.json in the `dist/*` folder.

Without this change, `dist/libs/payments/capability/package.json` will have value, `"main": "./src/index.js"` where the value should actually be `"main": "./libs/payments/capability/src/index.js"`.

As an alternative, what we could do is update the libs package.json, `libs/payments/capability/package.json`, with value `"main": "./libs/payments/capability/src/index.js"`. Some of the newer libraries created with Nx, already have this. (Albeit that they have the wrong value as well :P )